package.json

```diff
@@ -35,39 +36,44 @@
     "chalk": "^3.0.0",
     "debug": "^4.0.1",
     "ember-cli-version-checker": "^3.0.1",
-    "ember-template-lint": "^1.2.0",
+    "ember-template-lint": "github:ember-template-lint/ember-template-lint#next",
```

Comment:
Can you tweak to the released beta?

Reply:
It was failing on beta.4, I've released a beta.5 for template-lint